in_toto/util.py

```diff
@@ -103,7 +103,11 @@ def import_rsa_public_keys_from_files_as_dict(filepaths):
   key_dict = {}
   for filepath in filepaths:
     key = import_rsa_key_from_file(filepath)
-    keyid = key['keyid']
+    #FIXME: Add public key format check to ssl_crypto formats
+    if key["keyval"].get("private"):
```

Comment:
You are absolutely right. The thing is that [`KEYVAL_SCHEMA`](https://github.com/secure-systems-lab/securesystemslib/blob/master/securesystemslib/formats.py#L215-L218) requires us to always have a dict key named "private", even if it is empty.
I plan on submitting a PR for a `PUBKEYVAL_SCHEMA` that does what you propose, which would also replace this inline check.